actions/merger/entrypoint.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+
+set -e
+
+# config
+GITHUB_REPO=${GITHUB_REPO:-jade-data-repo}
+GITHUB_USER_EMAIL=${GITHUB_USER_EMAIL:-broadbot@broadinstitute.org}
+GITHUB_USER_NAME=${GITHUB_USER_NAME:-broadbot}
+GITHUB_WORKSPACE=${GITHUB_WORKSPACE}
```

Comment:
According to the github action docs, `GITHUB_WORKSPACE` is a default env var for a workflow. Are we assuming that we're going to get this setting from github, or as a user provided input? I'm worried that a user provided value could be overwritten by github when the action is run.

https://docs.github.com/en/actions/reference/environment-variables#default-environment-variables

Reply:
Good point - I removed it from the list of inputs. I think we just expect that it will be passed in from the action.